src/components/edit-hack.js

```diff
@@ -40,30 +40,30 @@ const useStyles = makeStyles(theme => ({
   marginFix: {
     marginLeft: theme.spacing(0.1),
     marginRight: theme.spacing(0.1),
-    paddingTop: theme.spacing(1),
+    paddingTop: theme.spacing(3),
     paddingBottom: theme.spacing(1)
   }
 }));
 
-export function EditHack({dispatch, hackDetails, history}) {
+export function EditHack({ dispatch, hackDetails, history }) {
   const classes = useStyles();
 
   const [updateData, setUpdateData] = useState({});
 
-
   const handleOnChange = e => {
     const { name, value } = e.target;
     setUpdateData({ ...updateData, [name]: value });
   };
 
   const onSubmitUpdate = e => {
     e.preventDefault();
-    dispatch(editingHackIdea(updateData,hackDetails._id))
-    dispatch(clearingHackDetails())
-    history.goBack()
+    dispatch(editingHackIdea(updateData, hackDetails._id));
+    dispatch(clearingHackDetails());
+    history.goBack();
   };
 
   if (hackDetails) {
+    console.log(updateData)
```

Comment:
pretty sure you dont want to keep this :)

Reply:
I mean I might want the user to see all that.
I has been removed